src/main/java/mate/academy/springboot/datajpa/dto/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import mate.academy.springboot.datajpa.dto.requestdto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.responsedto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper {
```

Comment:
you could use some abstraction for mappers (for example, two interfaces - one for request mapper and other for response)